lib/apiScenario/variableEnv.ts

```diff
@@ -262,6 +262,31 @@ export class VariableEnv {
     }
   }
 
+  public tryResolveString(source: string): string {
+    const regex = variableRegex;
```

Comment:
directly use `variableRegex` ?

Reply:
what's the difference with `doResolveStringVariable`?

Reply:
This function uses `get` not `getRequired`, it won't throw error when it not found. And only resolve `string` type without `bool` and `int`